server/modules/analysis/repository/calculation/update.js

```diff
@@ -0,0 +1,16 @@
+import { TableCalculation } from '@common/model/db'
+
+import { db } from '@server/db/db'
```

Comment:
let's start using relative imports in server/common/core

Reply:
ok